hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyDeleteRequestV1.java

```diff
@@ -149,9 +149,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       omBucketInfo = getBucketInfo(omMetadataManager, volumeName, bucketName);
 
       long quotaReleased = sumBlockLengths(omKeyInfo);
-      // update usedBytes atomically.
-      omVolumeArgs.getUsedBytes().add(-quotaReleased);
-      omBucketInfo.getUsedBytes().add(-quotaReleased);
+      omBucketInfo.incrUsedBytes(-quotaReleased);
```

Comment:
https://github.com/apache/ozone/pull/1607#discussion_r529974503
Pls refer this comment

Reply:
SumBlockLengths for OmKeyInfo here when it is directory it will be zero.

Reply:
Thanks a lot @bharatviswa504 for reminding this point again. Since this requires more deeper discussion I've created HDDS-4565 jira.  

Like we discussed in offline chat, it helps to unblock other PRs by fixing compilation errors in the branch(it occurred due to the recent rebase). I am changing the jira description as well reflecting the compilation error part.